data/mods/TropiCataclysm/monsters/tropical_insect_spider.json

```
@@ -41,47 +41,132 @@
+    "symbol": "l",
+    "color": "light_gray",
+    "volume": "5 L",
+    "weight": "6 kg",
+    "hp": 25,
+    "speed": 95,
+    "vision_night": 15,
+    "dodge": 2,
+    "harvest": "arachnid_firefly",
+    "reproduction": { "baby_egg": "egg_butterfly", "baby_count": 1, "baby_timer": 15 },
+    "upgrades": { "half_life": 14, "into": "mon_moth_giant" },
+    "baby_flags": [ "SPRING", "SUMMER" ],
+    "fear_triggers": [ "HURT", "FIRE" ],
+    "flags": [ "SEES", "HEARS", "FLIES", "STUMBLES" ]
+  },
+  {
+    "id": "mon_moth_giant",
+    "type": "MONSTER",
+    "name": { "str": "giant moth" },
+    "description": "A mutated moth larger than a human.  It silently flies with its huge wings during the night looking for nocturnal flowers to feed upon.",
+    "bodytype": "flying insect",
+    "species": [ "INSECT_FLYING" ],
+    "default_faction": "insect",
+    "material": [ "iflesh" ],
+    "symbol": "L",
+    "color": "light_gray",
+    "volume": "100 L",
+    "weight": "60 kg",
+    "hp": 50,
+    "speed": 95,
+    "vision_night": 15,
+    "dodge": 2,
+    "harvest": "arachnid_firefly",
+    "reproduction": { "baby_egg": "egg_butterfly", "baby_count": 1, "baby_timer": 15 },
+    "baby_flags": [ "SPRING", "SUMMER" ],
+    "fear_triggers": [ "HURT", "FIRE" ],
+    "flags": [ "SEES", "HEARS", "FLIES", "STUMBLES" ]
+  },
+  {
+    "id": "mon_caterpillar",
+    "type": "MONSTER",
+    "name": { "str": "caterpillar" },
+    "description": "Caterpillars are the larval stage of butterflies and moths.  This one seems newly hatched from an egg but even then its as large as a cat.",
+    "default_faction": "ant",
     "bodytype": "insect",
     "species": [ "INSECT" ],
-    "volume": "670000 ml",
-    "weight": "700000 g",
-    "hp": 100,
-    "speed": 80,
+    "volume": "30 L",
+    "weight": "40750 g",
+    "hp": 10,
+    "speed": 15,
     "material": [ "iflesh" ],
-    "symbol": "H",
-    "color": "yellow",
-    "aggression": 5,
-    "morale": 100,
-    "melee_skill": 5,
-    "melee_dice": 1,
-    "melee_dice_sides": 5,
-    "melee_cut": 0,
-    "dodge": 1,
-    "armor_bash": 6,
-    "armor_cut": 6,
-    "armor_bullet": 6,
-    "vision_day": 4,
-    "vision_night": 4,
-    "harvest": "arachnid_beetle",
-    "anger_triggers": [ "HURT", "MATING_SEASON" ],
-    "baby_flags": [ "AUTUMN" ],
-    "flags": [ "SEES", "SMELLS", "HEARS", "PUSH_MON", "PUSH_VEH" ]
+    "symbol": "c",
+    "color": "green",
+    "speed": 95,
+    "vision_day": 15,
+    "dodge": 2,
+    "harvest": "arachnid",
+    "upgrades": { "age_grow": 14, "into": "GROUP_BUTTERFLY" },
+    "flags": [ "SEES", "SMELLS", "CLIMBS", "HEARS", "STUMBLES", "PATH_AVOID_FIRE" ]
+  },
+  {
+    "id": "mon_caterpillar_giant",
+    "type": "MONSTER",
+    "name": { "str": "giant caterpillar" },
+    "description": "This caterpillar has reached the size of a person and it looks close to get into the cocoon phase.  You can only imagine the size of the insect that will come out of it.",
```

```suggestion
    "description": "This caterpillar has reached the size of a person and it looks close to reaching the pupa life stage.  You can only imagine the size of the insect that will emerge.",
```

See https://en.wikipedia.org/wiki/Pupa, I reworded it a bit so it looks a bit more smooth.